src/api/apiUtil.js

```diff
@@ -3,17 +3,18 @@ import { toast } from 'react-toastify';
 export const handleResponse = async (response) => {
   const data = await response.json();
   if (response.ok) {
-    await toast.success(response.ok);
+    await toast.success(data.message);
     return data;
   }
   if (data.status === 'error') {
-    const error = await response.text();
-    toast.success(data.message);
-    return error;
+    await toast.error(data.message);
+    return data;
   }
-  return 'Network response was not ok.';
+  await toast.error(data.message);
+  return data;
 };
```

Comment:
It would be nice to replace the many return statements with a single return. declare a variable using let and each time assign to it the condition's return value, then in turn return this variable, eg let res; if a; res=a; if b; res=b ...; 
return res;